### DISCUSSION TITLE
Question regarding custom music

### DISCUSSION THREAD

**Discussion by SolidFat** [IntelOrca/biorand#659] [Q&A] [answered]
Hello, I wanted to add in some Dino Crisis music to my game. I set up my custom music folder like the FAQ set it, but my music doesn't seem to play. Should the .wav files be named specific things for the game to find them, or is there a file limit size? They're all in properly named folders like the other BGM folders. Kinda curious what I've done wrong. Attatched some pics of my folder pathing.
![question2](https://github.com/IntelOrca/biorand/assets/123210039/f10ccece-36ca-48a7-b36d-2370f836a59e)
![question1](https://github.com/IntelOrca/biorand/assets/123210039/13df437f-9b82-4ec4-9891-b2a7473bc89b)

**Comment by IntelOrca** [comment, marked as answer]
Make sure the .wav files are 16-bit 44.4 Khz or 16-bit 22.05 Khz, alternatively try convering the files to .ogg. If that still doesn't work, try sending me one of the files.

**Reply by SolidFat** [reply]
Changing them to 16 bit worked! Thank you!
